src/main/java/com/synopsys/integration/jenkins/scan/SecurityScanner.java

```diff
@@ -37,18 +37,14 @@ public SecurityScanner(Run<?, ?> run, TaskListener listener, Launcher launcher,
         this.scannerArgumentService = scannerArgumentService;
     }
 
-    public int runScanner(Map<String, Object> scanParams) throws ScannerJenkinsException {
-        BlackDuckParametersService blackDuckParametersService = new BlackDuckParametersService(listener);
-
+    public int runScanner(Map<String, Object> scanParams, ScanStrategyService scanStrategyService) throws ScannerJenkinsException {
         BridgeDownloadParameters bridgeDownloadParameters = new BridgeDownloadParameters(workspace, listener);
         BridgeDownloadParametersService bridgeDownloadParametersService = new BridgeDownloadParametersService(workspace, listener);
         BridgeDownloadParameters bridgeDownloadParams = bridgeDownloadParametersService.getBridgeDownloadParams(scanParams, bridgeDownloadParameters);
 
```

Comment:
We should move `bridgeDownload` related operations and validation outside of this method. Maybe inside `ScanPipelineCommands` class.

Reply:
Moved `bridgeDownload` related operations and validation inside `ScanPipelineCommands` class